docs/contributing/Style-guide-for-Flutter-repo.md

```diff
@@ -1432,8 +1431,6 @@ Example:
     bar,
     baz,
   );
-  foo(bar,
-    baz);
```

Comment:
This is the only change I'm on the fence about.

Personally, I think the other two examples are much easier to visually parse than this one.

Reply:
Why are you on the fence with this one ? This would only make sense if it breaks the max char per line rule.

Reply:
If `foo()` takes up multiple lines due to a callback/switch expression/list or map literal, then I think it's all right to format it this way.

```dart
foo(bar, () {
  baz.quux();
  return baz.toInt();
});

foo(bar, switch (baz) {
  1 => 'a',
  2 => 'b`,
  _ => `c`,
});

foo(bar, [
  'quux',
  'qaax',
  'qeex',
]);
```

But in any other situation where the `foo()` arguments take up more than 1 line, I think it's much more readable with a trailing comma.

```dart
foo(
  bar,
  baz,
);
```

Reply:
I think it'd be nice to get another perspective on this, and then possibly add the above examples to this section.

Reply:
I agree with removing this example and would be fine with adding the other examples you're mentioning in the comment above.

Reply:
Awesome!

Examples have been added in the most recent commit. I'll plan on adding `autosubmit` tomorrow, to leave a window for any additional feedback 👍